integration-tests/gatsby-source-wordpress/__tests__/__snapshots__/index.js.snap

```diff
@@ -1493,7 +1493,6 @@ Array [
   Object {
     "fields": Array [
       "avatar",
-      "databaseId",
```

Comment:
This is because this field was already excluded in the test site but due to a bug that this PR happens to fix it's now properly excluded in the schema.